### ISSUE TITLE
Does not find/use existing configuration directory 

### ISSUE THREAD

**Issue by gingerbeardman** [myelin/rpcemu-macos#4]
On first launch I had to select my old RPCEmu directory at `~/Library/Application Support/RPCEmu/` — could your build maybe check for that and automatically use it if it exists?

**Comment by myelin** [comment]
It looks like this may require you to create a ~/Library/Application Support/RPCEmu folder -- details here: https://stardot.org.uk/forums/viewtopic.php?f=30&t=15360#p209354

**Comment by gingerbeardman** [comment]
What I mean is that I already had that directory from 0.8.x

Your 0.9.1 didn't seem aware of it. I had to manually locate it upon first launch. 

Given it is the standard location, and people will be upgrading from 0.8.x it makes sense to make the migration as smooth as possible?

**Comment by Septercius** [comment]
Technically speaking, it's not the standard location, because there were never any formal builds for the Mac - the 0.8.x ones were done by Theo or Francis and any 0.9.x ones have been done by me or myelin.  We had a long discussion on this topic on the StarDot forums (it got a bit heated, and was one of the reasons why I no longer post on that particular forum).  This was a sensible compromise to accommodate people who didn't like the stuff being in ~/Library, as they wanted to run different versions of the emulator/different versions of RISC OS.

**Comment by gingerbeardman** [comment]
Yes, I meant that it's as standard a place as macOS has for apps to store their own support files.

**Comment by gingerbeardman** [comment]
Still present in 0.9.1-pp3? untested
